libc/utils/MPFRWrapper/MPCommon.cpp

```diff
@@ -123,6 +123,21 @@ MPFRNumber MPFRNumber::atanh() const {
   return result;
 }
 
+MPFRNumber MPFRNumber::atanpi() const {
+  MPFRNumber result(*this);
+#if MPFR_VERSION >= MPFR_VERSION_NUM(4, 2, 0)
+  mpfr_atanpi(result.value, value, mpfr_rounding);
+  return result;
+#else
+  MPFRNumber value_atan(0.0, 1280);
+  mpfr_atan(value_atan.value, value, MPFR_RNDN);
+  MPFRNumber value_pi(0.0, 1280);
```

Comment:
`mpfr_precision * 3` is enough here. Reduces the exhaustive test run time from 2 * ~3s to 2 * ~0.4s on my machine (Intel Core i7-13700H).

```suggestion
  MPFRNumber value_atan(0.0, mpfr_precision * 3);
  mpfr_atan(value_atan.value, value, MPFR_RNDN);
  MPFRNumber value_pi(0.0, mpfr_precision * 3);
```